ivy/func_wrapper.py

```diff
@@ -1580,6 +1580,38 @@ def func(x):
     return _handle_backend_invalid
 
 
+def _warn_efficient_implementation_available(fn: Callable) -> Callable:
+    @functools.wraps(fn)
+    def _wrapper(*args, **kwargs):
```

Comment:
it would be great to name the wrapper function and the wrapped function in the same way as the other function wrappers